### DISCUSSION TITLE
Problem with the native installation for octomap

### DISCUSSION THREAD

**Discussion by dorian21b** [ctu-mrs/mrs_uav_system#166] [Q&A]
Hello

I posted an issue on the octomap_mapping_planning about a problem I am facing:
https://github.com/ctu-mrs/octomap_mapping_planning/issues/10

I posted this here in case anyone can help me or if anyone have the same problem. 
Thank you for time.

Best, 
Dorian

**Comment by klaxalk** [comment]
Hey, with the [new system](https://github.com/ctu-mrs/mrs_uav_system) out, you can install our [Octomap Mapping Planning](https://github.com/ctu-mrs/mrs_octomap_mapping_planning) package already pre-compiled and working.
```bash
curl https://ctu-mrs.github.io/ppa-stable/add_ppa.sh | bash
sudo apt install ros-noetic-mrs-uav-system-full
```
